etc/workflows/partial-publish.xml

```diff
@@ -10,6 +10,17 @@
 
   <operations>
 
+    <operation
+        id="select-tracks"
+        exception-handler-workflow="partial-error"
+        description="Preparing presenter (camera) audio and video work versions">
+      <configurations>
+        <configuration key="source-flavor">*/prepared</configuration>
+        <configuration key="target-flavor">*/work</configuration>
```

Comment:
So `/work` is used in this partial, and no where else?

Reply:
True. I guess we could rename it, but I couldn't think of something better...

Reply:
Derp, I missed that this was used as source for other things in this partial.  I was expected it to be used in *other* partials...